### ISSUE TITLE
Unable to use arrow keys

### ISSUE THREAD

**Issue by icedcoffeeee** [galatolofederico/manim-presentation#4]
I followed all the instructions on the readme, and everything seems to be working fine, all keybinds are okay, space, R, Q, all of them except the arrow keys. Is this a problem of the module or is this unrelated at all?
Additional Info: I'm using Windows 10 64x

**Comment by icedcoffeeee** [comment]
nvm, i changed the local package source code to other letters

**Comment by galatolofederico** [comment]
Yeah i really need to spend 10 minutes implementing a keybind configuration file.
Can you please copy your config here so i can use your key numbers as default on windows?

**Comment by icedcoffeeee** [comment]
Oh I see.. i tried to find the actual numbers, but turns out they came in doubles. "\xe0","M" for the right arrow key (based on getch() method from msvcrt package). Since that made things quite difficult, instead, i remapped them to my square brackets.
```py
class Config:
    QUIT_KEY = ord("q")
    CONTINUE_KEY = ord("]")        #right arrow
    BACK_KEY = ord("[")            #left arrow
    REWIND_KEY = ord("r")
    PLAYPAUSE_KEY = 32     #spacebar
```
Certainly not a solution but functional nonetheless. Thanks for the plugin! :)

**Comment by icedcoffeeee** [comment]
UPDATE: i got it to work with the arrow keys, these are my changes to the `present.py` file:
```py
class Config:
    QUIT_KEY = ord("q")
    CONTINUE_KEY = 2555904      #right arrow
    BACK_KEY = 2424832          #left arrow
    REWIND_KEY = ord("r")
    PLAYPAUSE_KEY = 32     #spacebar
```
and a tweak at line 193
```py
key = cv2.waitKeyEx(fix_time(sleep_time - self.lag))
```
thanks again!

**Comment by galatolofederico** [comment]
It is a bit dirty but for now 0919977 should solve this issue. I will wait to address the other open issues and then i will merge everything to main

**Comment by galatolofederico** [comment]
Solved in 2eafa0b82e7d3354628354bc084736b953331a86
